content/pages/resources.md

```diff
@@ -0,0 +1,29 @@
+Title: Resources
+Category: pages
+Summary: Feminism101 and Other Resources
```

Comment:
I would add a Status: hidden on this page, then we can link to it from our FAQ. Otherwise, it will show up on the nav bar and push things off the page.